### PR TITLE
NFDIV-1127: Set up cron job

### PR DESCRIPTION
**Pull request by tomxelliott**

### CODE REVIEW

File: k8s/namespaces/nfdiv/nfdiv-cron-applicant1-apply-conditional-order/nfdiv-cron-applicant1-apply-conditional-order.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+              alias: S2S_SECRET
+            - name: idam-secret
+              alias: IDAM_CLIENT_SECRET
+            - name: idam-systemupdate-username
+              alias: IDAM_SYSTEM_UPDATE_USERNAME
+            - name: idam-systemupdate-password
+              alias: IDAM_SYSTEM_UPDATE_PASSWORD
+      environment:
+        TASK_NAME: SystemNotifyApplicant1ApplyForConditionalOrder
+      schedule: 0/10 * * * *
```
Comment (nitinprabhu) [inline k8s/namespaces/nfdiv/nfdiv-cron-applicant1-apply-conditional-order/nfdiv-cron-applicant1-apply-conditional-order.yaml:31]: I think this schedule format was changed by platops I think it was `*/10 * * * *` @linusnorton  ?

Comment (tomxelliott) [reply k8s/namespaces/nfdiv/nfdiv-cron-applicant1-apply-conditional-order/nfdiv-cron-applicant1-apply-conditional-order.yaml:31]: ok will change, just followed the approach taken with the other jobs.

Comment (tomxelliott) [reply k8s/namespaces/nfdiv/nfdiv-cron-applicant1-apply-conditional-order/nfdiv-cron-applicant1-apply-conditional-order.yaml:31]: That format doesn't appear to be valid yaml @nitinprabhu

Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-cron-applicant1-apply-conditional-order/nfdiv-cron-applicant1-apply-conditional-order.yaml:31]: I got confused I think

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**